recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
I'd rather come up with something at conda-forge to handle these sorts of things automatically. Be it through `MAKEFLAGS` or some other medium. It would be really nice to have these conversations in one place around one thing. Though we can't do this now because we are handling it in each recipe one at a time.